docs/java/java-tutorial.md

```diff
@@ -59,21 +60,21 @@ For this tutorial, the only required extensions are:
 
 The supported version for running the VS Code for Java extension and the supported version for your projects are two separate runtimes. To run VS Code for Java, Java SE 11 or above version is required; for projects, VS Code for Java supports projects with version 1.5 or above. For more details, refer to [Configure JDK](/docs/java/java-project.md#configure-jdk).
```

Comment:
There doesn't seem to be a "VS Code for Java extension". Was this perhaps an old name for the Java extension pack, or something?

For the record, I've also left a comment to this effect in #4003, which is where this language was first introduced.

Reply:
yes, we should probably make it "Extension pack for Java" for consistency

Reply:
Ok, will change!

Reply:
Actually, on a closer look, I'm wondering if we could be a little more specific. I mean, the extension pack itself doesn't require Java 11. This seems to be specifically related to the language server, which IIUC is provided by the [Language Support for Java](https://marketplace.visualstudio.com/items?itemName=redhat.java) extension (which is included in the extension pack). Indeed, that page says exactly this:

> (latest Java 11 is the current minimum requirement).

This is consistent with what's written in [java-project.md](https://github.com/microsoft/vscode-docs/blob/main/docs/java/java-project.md):

> Although the Java language server requires JDK version 11 or above to run, this is NOT a requirement to your project's runtime.

It also matches what the configuration wizard says:

![Screenshot from 2021-08-13 18-21-59_](https://user-images.githubusercontent.com/478237/129397341-7bdae0d5-2625-4dfc-aa9e-348e241097ac.png)

(Although clearly the second message is too ambiguous and should be fixed too — I've submitted https://github.com/microsoft/vscode-java-pack/issues/729 to track that.)

So, maybe we should make this a reference to the `redhat.java` extension (in whatever phrasing is appropriate here), rather than to the extension pack?

Reply:
I'm merging this PR since it has a lot of great edits. If you want to add clarification here, please open another PR.

Reply:
Good point. This was not part of the PR's diff, and probably warrants discussion in a separate issue.

Reply:
Ok, I've opened #4786, and will mark this conversation as resolved, to keep this PR's thread focused on the actual changes made here.